devices/tuya/_TZE200_t1blo2bj_neo_siren.json

```diff
@@ -0,0 +1,231 @@
+{
+  "schema": "devcap1.schema.json",
+  "uuid": "78685bfe-85cf-462a-bff9-059a876ffbb0",
+  "manufacturername": [
+    "_TZE200_t1blo2bj",
+    "_TZE204_t1blo2bj"
+  ],
+  "modelid": [
+    "TS0601",
+    "TS0601"
+  ],
+  "product": "Neo (Tuya) smart siren",
+  "sleeper": true,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_ON_OFF_LIGHT",
+      "restapi": "/lights",
+      "uuid": [
+        "$address.ext",
+        "0x01"
+      ],
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/swversion",
+          "refresh.interval": 86400,
+          "parse": {
+            "fn": "zcl:attr",
+            "ep": 1,
+            "cl": "0x0000",
+            "at": "0x0001",
+            "script": "tuya_swversion.js"
+          },
+          "read": {
+            "fn": "zcl:attr",
+            "ep": 1,
+            "cl": "0x0000",
+            "at": "0x0001"
+          }
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/tuya_unlock",
+          "public": false,
+          "refresh.interval": 86400,
+          "default": false
+        },
+        {
+          "name": "state/alert",
+          "default": "none"
+        },
+        {
+          "name": "state/on",
+          "write": {
+            "dpid": 13,
+            "dt": "0x10",
+            "eval": "Item.val == 1 ? 1 : 0;",
+            "fn": "tuya"
+          },
+          "parse": {
+            "dpid": 13,
+            "eval": "Item.val == 1 ? 0 : 1;",
+            "fn": "tuya"
+          },
+          "default": false
+        },
+        {
+          "name": "state/reachable"
+        }
+      ]
+    },
+    {
+      "type": "ZHAAlarm",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0500"
+      ],
+      "fingerprint": {
+        "profile": "0x0104",
+        "device": "0x0403",
```

Comment:
The device ID is incorrect here

Reply:
hi @SwoopX where do I find the correct device ID?

![image](https://github.com/dresden-elektronik/deconz-rest-plugin/assets/5397828/ed6ba1b5-4d9a-414d-a365-076784495264)

Reply:
The device id 0x0403 is for IAS_WARNING_DEVICE
You can see it on the node title, your is 0x0051 for "Smart plug"

Reply:
Thank you! I updated the file.